src/Sylius/Bundle/ShippingBundle/spec/Form/Type/RuleChoiceTypeSpec.php

```diff
@@ -22,8 +22,8 @@
 class RuleChoiceTypeSpec extends ObjectBehavior
 {
     private $choices = [
-        RuleInterface::TYPE_ITEM_TOTAL => 'Order total',
-        RuleInterface::TYPE_ITEM_COUNT => 'Order items count',
+        RuleInterface::TYPE_UNIT_TOTAL => 'Order total',
```

Comment:
Yeah I was not sure about this, for both the TOTAL and the COUNT. But fundamentally it _is_ units. The TYPE_UNIT_TOTAL rule doesn't appear to have any use, but TYPE_UNIT_COUNT certainly does. If you look down at the `PerUnitRateCalculator` `calculate()` method:

```
return (int) ($configuration['amount'] * $subject->getShippingUnitCount());
```

Previously this was `getShippingItemCount()` but it was of course getting the count of `OrderItemUnits` so effectively this is correct to call it `TYPE_UNIT_COUNT`.

I was thinking that you might want to add a CoreBundle extra rule which is TYPE_ITEM_COUNT and works on an OrderItem basis, but for now this is at least accurate to what is being counted?